Valet/Valet iOS-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
```

Comment:
we don't need this file unless we think objc is going to pull in swift stuff, which seems unlikely.